_includes/navbar.html

```diff
@@ -3,11 +3,14 @@
 
 <nav id="header">
   <span id="left-side">
-    <a href="{{site.baseurl}}/" class="navbar_tab">Home</a>
-    <a href="{{site.baseurl}}/main/" class="navbar_tab">ACLE</a>
-    <a href="{{site.baseurl}}/mve_intrinsics/" class="navbar_tab">MVE</a>
-    <a href="{{site.baseurl}}/morello/" class="navbar_tab">Morello</a>
-    <a href="{{site.baseurl}}/neon_intrinsics/" class="navbar_tab">Neon</a>
+    {% for nav in site.data.navigation %}
```

Comment:
Can you add a comment saying that the for loop is looping on the content of ` _data/navigation.yml`? It might be pretty clear for people used to Jekyll/Liquid, but not for outsiders.